### PR TITLE
Platform left nav

### PR DESCRIPTION
**Pull request by ankit1khare**
Pages revamped:
[Device-level Experiments](https://docs.statsig.com/guides/first-device-level-experiment)
[Custom ID Types](https://docs.statsig.com/guides/experiment-on-custom-id-types)
[Environments](https://docs.statsig.com/guides/using-environments)
[Testing Your Setup](https://docs.statsig.com/guides/testing)
and FAQ

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://66fcfe7c537e8d9186f92a80--cozy-fox-0defec.netlify.app

File: docs/guides/experiment-on-custom-id-types.mdx [hunk 1]
```diff
@@ -3,47 +3,83 @@ sidebar_label: Custom ID Types
...
 
-Log into the Statsig console at https://console.statsig.com/ and navigate to Project Settings in top-right panel, and then find "Custom Unit IDs" section under the "Basic Settings" tab.
+> **Note**: These steps also apply to feature gates, allowing you to partially roll out features based on custom Unit IDs.
 
-![Screen Shot 2022-06-26 at 10 53 06 AM](https://user-images.githubusercontent.com/74584483/175827590-147ff9c8-ca02-4298-bca5-96583904aec5.png)
+---
+
+### Step 1: Add `companyID` as a Custom Unit ID
+
+1. **Log into the Statsig Console**: Head over to [Statsig Console](https://console.statsig.com/) and navigate to **Project Settings** in the top-right corner of the dashboard.
```
Comment (tore-statsig) [inline docs/guides/experiment-on-custom-id-types.mdx:23]: its in the bottom left now - screnshots are wrong, but so are the directions

File: docs/guides/experiment-on-custom-id-types.mdx [hunk 2]
```diff
@@ -3,47 +3,83 @@ sidebar_label: Custom ID Types
...
+3. **Add a New Custom Unit ID**:
+    - Click the **Edit** button.
+    - Enter `companyID` as the new ID type and provide a description.
+    - Save the changes.
 
-![Screen Shot 2022-06-26 at 10 56 17 AM](https://user-images.githubusercontent.com/74584483/175827554-8862e10a-4171-4028-9aa4-9f314718b39d.png)
+    ![Add Custom ID](https://user-images.githubusercontent.com/74584483/175827554-8862e10a-4171-4028-9aa4-9f314718b39d.png)
 
-Once the new ID type is added, you can use it for all new experiments going forward - no need to repeat this step again!
+Once added, this custom Unit ID can be used across all your experiments. You only need to configure it once for your project!
```
Comment (tore-statsig) [inline docs/guides/experiment-on-custom-id-types.mdx:36]: across all your statsig configurations: experiments, gates, layers, dynamic configs, autotunes, etc

File: docs/guides/featureflags-or-experiments.md [hunk 3]
```diff
@@ -4,39 +4,62 @@ title: When to use Feature Flags vs. Experiments?
...
 
-Usually referred to online as _feature flags_, the Statsig UI and SDKs call them _feature gates_.
+With Statsig, you can create control and test groups and compare them using A/B testing via either **feature gates** or **experiments**. This guide helps clarify when to use a feature gate versus an experiment, depending on your needs.
 
-:::
+---
+
+## What Do You Need?
+
+- **Feature Gate**: Use this if you want to measure the impact of a feature and roll it out gradually.
```
Comment (tore-statsig) [inline docs/guides/featureflags-or-experiments.md:16]: or just for rollouts, if you dont care about measurement

File: docs/guides/featureflags-or-experiments.md [hunk 4]
```diff
@@ -4,39 +4,62 @@ title: When to use Feature Flags vs. Experiments?
...
- - A Feature Gate exposure check returns a boolean (true/false) to assign the user to either the control or test group; an Experiment exposure check returns a JSON config to help you configure your app experience based on the group that the user is assigned to
- - To ramp up, you will increase the Pass% in a Feature Gate, and increase the Allocation% in an Experiment. In both cases, users in each group will continue to receive their first assigned experience i.e. ramping up Pass% or Allocation% does not resalt (or reshuffle) users. 
+2. **Exposure Results**:
+   - **Feature Gate**: Returns a boolean (`true` or `false`) to assign users to the control (off) or test (on) group.
+   - **Experiment**: Returns a JSON config that helps configure your app experience for the user's assigned group.
+
+3. **Ramping Up**:
+   - **Feature Gate**: Increase the **Pass%** to gradually roll out a feature to more users.
+   - **Experiment**: Increase the **Allocation%** to add more users to your test.
```
Comment (tore-statsig) [inline docs/guides/featureflags-or-experiments.md:36]: you can go up to 99 vs 1 percent with fg, you cant go past 50/50 with expeirments

File: docs/guides/first-device-level-experiment.mdx [hunk 5]
```diff
@@ -3,62 +3,84 @@ sidebar_label: Device-level Experiments
 title: Build your first Device-level Experiment
 ---
 
-In cases where you're unable to establish the user's identity and are unable to leverage a user ID as the unit of randomization for your experiment,
-Statsig's _device-level_ experiments enable you to design experiments using a Statsig-generated stable ID, saved on the user's device in local storage.
+When you cannot identify a user via their user ID, device-level experiments allow you to randomize experiments based on a _stable ID_ stored locally on the user's device. This stable ID is automatically generated by Statsig and ensures consistent evaluations across sessions for anonymous users.
```
Comment (tore-statsig) [inline docs/guides/first-device-level-experiment.mdx:6]: I feel like we should push people to use their own cookie/logged out id, and only rely on stableid if they have nothing

File: docs/guides/first-device-level-experiment.mdx [hunk 6]
```diff
@@ -3,62 +3,84 @@ sidebar_label: Device-level Experiments
...
 
-Log into the Statsig console at https://console.statsig.com/ and navigate to Experiments+ in the left-hand navigation panel.
+### Step 1: Create a Device-level Experiment
 
-Click on the Create button and enter the name and description for your experiment. Click the checkbox for **Use Stable ID** check box and click **Create**.
+1. **Log into the Statsig Console**: Visit [Statsig Console](https://console.statsig.com/) and navigate to **Experiments+** on the left-hand sidebar.
+   
+2. **Create a New Experiment**:
+    - Click on **Create** and fill out the **name** and **description** of your experiment.
+    - Enable the **Use Stable ID** option during setup.
```
Comment (tore-statsig) [inline docs/guides/first-device-level-experiment.mdx:20]: thats not a thing?  You need to select the unitID as stableID

**Review by tore-statsig — APPROVED** [review]

**Pull request merged**